### ISSUE TITLE
Invalid references in twisted.mail

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8516]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)| @acabhishek942 reported|
|-|-|
|Trac ID|trac#8516|
|Type|enhancement|
|Created|2016-06-27 18:10:17Z|

Following are the invalid references in twisted.mail obtained by checking out the trunk and running

```
./bin/admin/build-apidocs . apidocs
```

```
twisted.mail.smtp.sendmail: invalid ref to email.Generator
twisted.mail.smtp.sendmail: invalid ref to email.Message
```

<details><summary>Searchable metadata</summary>

```
trac-id__8516 8516
type__enhancement enhancement
reporter__acabhishek942 acabhishek942
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1467051017566760 1467051017566760
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_1"></a>|@acabhishek942 commented|
|-|-|

See : https://github.com/twisted/twisted/pull/239

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset ed9ba696f733a0d2bbf818bd05eae1fe24360368
```
#!CommitTicketReference repository="" revision="ed9ba696f733a0d2bbf818bd05eae1fe24360368"
Merge 8516-acabhishek942-doc: Fix invalid references in twisted.mail

Author: acabhishek942
Reviewer: rodrigc
Fixes: #8516
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
